app/models/katello/host_collection.rb

```diff
@@ -30,8 +30,13 @@ class HostCollection < Katello::Model
     scoped_search :on => :organization_id, :complete_value => true
 
     def validate_max_content_hosts
+      if new_record?
```

Comment:
I think using `on: :create` is more readable:

```ruby
validates :systems, length: {maximum: -> (record) { record.max_content_hosts }, 
  if: -> (record) { record.unlimited_content_hosts }, on: :create
```

However, not sure this is more readable though than just creating a `validate` method...